### PR TITLE
master: replace InstallSchema.php with db_schema.xml and bumped version

### PR DESCRIPTION
**Pull request by MeneerHaas**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "clang/module-clang",
     "description": "A module to integrate Magento with CanopyDeploy e-mailmarketing software",
-    "version": "1.0.13",
+    "version": "1.0.14",
```
Comment (jeroen-de-bruin) [inline composer.json:4]: Er is nog een composer json waar dit moet worden aangepast :-)

File: Clang/RELEASENOTES [hunk 2]
```diff
@@ -1,5 +1,13 @@
 Release notes
```
Comment (jeroen-de-bruin) [inline Clang/RELEASENOTES:1]: Er moet ook nog een zip file worden toegevoegd.

**Review by jeroen-de-bruin — CHANGES_REQUESTED** [review]

**Review by jeroen-de-bruin — APPROVED** [review]

**Review by maarten-de-reus — APPROVED** [review]

**Comment by leonhelmus** [comment]
Hey I would like to add an issue, but the tab for issues is not available is there a reason why this is not enabled? I had some troubles with 2.4.3-p2 and it does not seem to work on that version anymore.

I wanted to add steps to reproduce.

**Comment by jeroen-de-bruin** [comment]
> Hey I would like to add an issue, but the tab for issues is not available is there a reason why this is not enabled? I had some troubles with 2.4.3-p2 and it does not seem to work on that version anymore.
> 
> I wanted to add steps to reproduce.

Hi, thanks for your message. Please contact support if you have issues, we do not use this channel for communication.

**Pull request merged**